README.md

```diff
@@ -17,48 +17,31 @@ Once in a while, it connects to APIs to get the latest information about session
 It then augments and republishes this information for clients to consume. 
 It also provides a couple of extra APIs to save your favorites and accumulate votes.
 
-### Android Application
+### iOS and Android Applications
 
-As you can imagine, the Android version is developed in Kotlin/JVM. What's interesting here is that this time
-application utilizes Multiplatform support, which is an experimental feature in Kotlin 1.3.61. Data structures for retrieving data from the backend server and some date-time operations are shared across multiple projects.
-
-### iOS Application
-
-User interface of iOS version is written in Swift, all logic and data written in Kotlin in the common module. 
-This way iOS part itself is responsible only for specifying how the application looks like and how it represents changes requested by logic. Kotlin and Swift are highly interoperable, so from Swift, you can easily use all classes and tools defined in the common module. 
+Both Android and iOS applications are developed using Kotlin Multiplatform Mobile technology within a single codebase.
+The UI is implemented using Compose Multiplatform library, sharing the same code between platforms.
 
 ## How to build and run
 
-### Building the code
+### Pre-requisites
 
  * Make sure you have the Android SDK installed
- * Open the project in IntelliJ IDEA (2019.3 recommended)
  * Create a file `local.properties` in the root directory of the project, pointing to your Android SDK installation. On Mac OS, the contents should be `sdk.dir=/Users/<your username>/Library/Android/sdk`. On other OSes, please adjust accordingly.
- * Run `./gradlew build`
-
-### Running the backend
- 
- * Run `./gradlew backend:run` from the command line or from Gradle tool window
- * The backend will start serving on localhost:8080, with data stored in a local H2 database
 
 ### Running the Android app
 
- * Create a run configuration of type "Android App"
- * Select module "app" in the run configuration settings
- * Run the configuration
- * Select the emulator or connected device, as normal
+* Open project in Android Studio or JetBrains Fleet: https://www.jetbrains.com/fleet/
```

Comment:
- Android Studio and Fleet should go in prerequisites.
- Here you can also use numbered steps.

Edit:
 1. Open the project in Android Studio or JetBrains Fleet and wait until the project finishes loading.